core/src/main/kotlin/fr/sncf/osrd/stdcm/infra_exploration/InfraExplorer.kt

```diff
@@ -145,8 +148,12 @@ private class InfraExplorerImpl(
     private var incrementalPath: IncrementalPath,
     private var pathPropertiesCache: MutableMap<BlockId, PathProperties>,
     private var currentIndex: Int = 0,
-    private val endBlocks:
-        Collection<BlockId>, // Blocks on which "end of path" should be set to true
+    // TODO: Should evolve into a List of Collection of struct that contains location, duration and
+    // onStopSignal
+    private val stops:
+        List<
+            Collection<PathfindingEdgeLocationId<Block>>
+        >, // Blocks on which "end of path" should be set to true
```

Comment:
```suggestion
        >,
```

Outdated

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/7322/commits/1c9bce5bfc2c52375414a9b6f25297b7dba021e5